packages/plugin-zod/src/components/Operations.tsx

```
@@ -33,8 +33,28 @@ export function Operations({ name, operations }: Props) {
+  };
+  readonly responses: {
+        readonly [status: number]: z.ZodTypeAny | undefined;
+        readonly default: z.ZodTypeAny | undefined;
+  };
+  readonly errors: {
+        readonly [status: number]: z.ZodTypeAny | undefined;
+  };
+}`}</Type>
+      </File.Source>
+      <File.Source name='OperationsMap' isExportable isIndexable>
+        <Type name='OperationsMap' export>{`Record<string, OperationSchema>`}</Type>
```

Biome (2.1.2)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Safe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)